controllers/workspace/devworkspace_controller.go

```diff
@@ -120,6 +121,7 @@ func (r *DevWorkspaceReconciler) Reconcile(req ctrl.Request) (reconcileResult ct
 		workspace.Status.Phase = dw.DevWorkspaceStatusStarting
 		workspace.Status.Message = "Initializing DevWorkspace"
 		err = r.Status().Update(ctx, workspace)
+		metrics.WorkspaceStarted(workspace, reqLogger)
```

Comment:
Should not we check error before incrementing Started?

Reply:
Won't it increment Started when creating devworkspace with Started: false.

I'm afraid I don't understand why we even set Starting phase here, I would say we just should update status with DevWorkspace, and STOPPED.

Reply:
You're right, I don't know what I was thinking doing it this way. I'll fix this before merging, but I got side-tracked with debugging https://github.com/devfile/devworkspace-operator/pull/527 today.

Reply:
Should be fixed, though there's still a bit of strangeness in having to set the Started condition ASAP and then also having to do
```
	reconcileStatus := currentStatus{phase: dw.DevWorkspaceStatusStarting}
	reconcileStatus.setConditionTrue(conditions.Started, "DevWorkspace is starting")
```
to ensure the condition isn't wiped out later.